### DISCUSSION TITLE
"BG reading are unavailable" instead of chart on notification window

### DISCUSSION THREAD

**Discussion by jgr-lab** [NightscoutFoundation/xDrip#2218] [Q&A] [answered]
Hi. 
I am using dexcomm G6 with BYODA app, xiaomi 4x with android 11 (lineageOS). BYODA app sends BG data to AndroidAPS app (AAPS.) AAPS sends data to xdrip and NS, but xdrip displays notification "BG reading are unavailable" instead of BG chart. After opening xdrip I can see correct data. All apps are have battery optimization disabled. On attached screenshots you can see that notification shows that last reading was 4minutes ago but it can't display chart. Alarms also don't work with this setting.

I am using xdrip version e557cba, but it also occurred on older versions. 
I used older version on my previous 2 phones successfully with this setup

![Screenshot_20220711-165242_AndroidAPS](https://user-images.githubusercontent.com/9194546/178295175-43165e4e-8e79-46d6-9520-aac5372fc528.png)

Xdrip settings:
![Screenshot_20220711-165301_xDrip+](https://user-images.githubusercontent.com/9194546/178295181-fc9f6ca0-ab1d-4521-81ce-56b519b4d689.png)
![Screenshot_20220711-165328_xDrip+](https://user-images.githubusercontent.com/9194546/178295185-730bdcbd-1dcf-467b-882e-ec6fd61b9474.png)
AAPS settings::
![Screenshot_20220711-165356_AndroidAPS](https://user-images.githubusercontent.com/9194546/178295186-a3eb3715-e058-49aa-9ec0-cab94f991ddc.png)

**Comment by Navid200** [comment]
Did you set up the app to transmit to xDrip as well?  See the image please.
![Screenshot 2022-07-11 112833](https://user-images.githubusercontent.com/51497406/178300730-9f594862-61bb-4dbb-aace-308dc24e348b.png)

**Reply by jgr-lab** [reply]
To be honest I don't remember, but this is my backup plan. If nothing helps then during next sensor change I will reinstall BYODA.
Nevertheless note that:
1) exactly the same apk file worked on my previous 2 phones
2) xdrip in my configuration is getting BG data from AAPS

**Reply by tolot27** [reply]
> 1. exactly the same apk file worked on my previous 2 phones

That's not in line with your first post:

> I am using xdrip version https://github.com/NightscoutFoundation/xDrip/commit/e557cba811df35ccdeea38801ab9df5e782f7c5a, but it also occurred on older versions.


Here you are referring to a commit. Please can you link to the release/nightly you are using?

**Reply by jgr-lab** [reply]
You are right. So at the begging I tried to use my old apk file which used to work. After I couldn't make it work I downloaded newest version available by in app update button.



> Here you are referring to a commit. Please can you link to the release/nightly you are using?

How to do it? As I said I download new version of apk through xdrip itself. I couldn't find any other versioning information

**Reply by tolot27** [reply]
The version is shown at the first system status page and also the [release page](https://github.com/NightscoutFoundation/xDrip/releases).

**Comment by tolot27** [comment, marked as answer]
And just another question: What does the burger menu shows? "Start Sensor" or "Stop Sensor". If it shows "Start Sensor", please start it.

**Reply by jgr-lab** [reply]
Looks like pressing "start sensor" helped. What happened?
I'll test tomorrow if everything works fine, in the last few days I had problems with randomly blocked reading to AAPS when xdrip was working
